README.md

```diff
@@ -34,9 +34,13 @@ python sum.py -i sample.mp4 -s subtitle.srt
 ```
 To download a YouTube video along with its subtitles from url:
 ```python
-python sum.py -u "url"
+python sum.py -u <url>
```

Comment:
Need to mention that the downloaded files will remain in this case 👍

Reply:
Only with -k, the downloaded files will be kept.